app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js

```diff
@@ -137,8 +137,10 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
     }
 
     function buildImageId(image) {
-      if (image.fromContext) {
+      if (image.fromContext && image.pattern && image.cluster) {
```

Comment:
Yeah it's meant to cover the findImage case. I agree, it's good to be clear here.